src/views/ConfigEditor/FormParts/DefaultValues/DefaultValues.api.tsx

```diff
@@ -1,19 +1,71 @@
 import {getBackendSrv} from '@grafana/runtime';
 
 
-export const getOptions = (query: string, url: string): Promise<any> => {
+export const getOptions = async (query: string, url: string, options: any): Promise<any> => {
+  const _getRequestOptions = (query: string, usePOST?: boolean, requestId?: string) => {
+    let requestOptions: any = {
+      url: options.access === 'proxy' ? `/api/datasources/proxy/uid/${options.uid}` : options.url,
+      requestId: requestId,
+    };
+    let params: string[] = [];
+
+    if (usePOST) {
```

Comment:
Why we copy\paste here instead of extract the common code in `datasource.ts` and re-use it?